tests/io/test_binary.cpp

```diff
@@ -29,7 +29,7 @@ TEST(io_binary, cell) {
 
     // Set event configuration
     const std::size_t event = 0;
-    const std::string cells_directory = "tml_full/ttbar_mu200/";
+    const std::string cells_directory = "tml_full/ttbar_mu100/";
```

Comment:
Is this meant to be in this PR?

Reply:
OK, nvm. I did not read your comment

Reply:
As I wrote above, I came across a race condition in the CI.

Those "binary tests" are writing temporary files into the data directory. (Which is pretty bad, as the data directory should really be treated as read-only...) And both the "cell" and "measurement" test write a file with the cell module information. With the same name. So under just the right circumstances the two can overwrite each others' files. (The two tests write exactly the same file. So for the error to appear, they have to ask the OS to flush their writes exactly at the same time in some weird way.)

Now... I could've opened a separate PR for the test fix. But since it was affecting the CI of this PR, I thought it would be fine to just slap it in here. (Since the fix/workaround is just 2 lines in the end.)